static/js/github-create-branch.js

```diff
@@ -118,6 +122,20 @@ $(document).ready(() => {
   });
 });
 
+
+const validateSourceBranch = (branchName) => {
+	hideValidationErrorMessage("ghParentBranch");
+	const repo = getRepoDetails();
+	const url = `/github/branch/owner/${repo.owner}/repo/${repo.name}/${branchName}`;
+
+	$.get(url)
+		.fail((error) => {
```

Comment:
Only interested in 404
- Success ignore (potentially expand to give a tick or visual validation in future story)
 - all other 4**/5** are ignored for this check, we only care if we try and cant find it, other issue handled later